client/signup/config/flows.js

```diff
@@ -135,6 +135,13 @@ const flows = {
 		lastModified: null
 	},
 
+	'delta-new': {
+		steps: [ 'themes', 'domains', 'plans', 'user' ],
+		destination: getCheckoutDestination,
```

Comment:
We _just_ changed the destination function names at the top of this file, so you'll want to rebase with master and change this line to match the new destination of the main flow:

```js
destination: getSiteDestination,
```